### PR TITLE
FE-900 - Component Wrapper for Matchbox ScreenReaderOnly

### PR DESCRIPTION
**Pull request by nicklemmon**
[FE-900](https://jira.int.messagesystems.com/browse/FE-900)

### What Changed
- Adds component wrapper for Matchbox `<ScreenReaderOnly/>`
- Replaces direct imports from Matchbox with app component
- Adds supporting unit test
- Uses `as` prop to pass in `<div>` for situations in which the the `<ScreenReaderOnly>` component wraps a block level element

### How To Test
- Verify `<ScreenReaderOnly/>` continues to render children to the DOM and does not render the children visually

### To Do
- [ ] Incorporate feedback

### CODE REVIEW

File: src/components/fullPage/FullPage.js [hunk 1]
```diff
@@ -1,44 +1,41 @@
 import React from 'react';
-import { UnstyledLink, ScreenReaderOnly } from '@sparkpost/matchbox';
+import { UnstyledLink } from '@sparkpost/matchbox';
 import { ChevronLeft } from '@sparkpost/matchbox-icons';
 import propTypes from 'prop-types';
+import { ScreenReaderOnly } from 'src/components/matchbox';
```
Comment (nicklemmon) [inline src/components/fullPage/FullPage.js]: Just changed this - the rest is Prettier.

File: src/components/iconButton/IconButton.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import classNames from 'classnames';
 import PropTypes from 'prop-types';
-import { ScreenReaderOnly } from '@sparkpost/matchbox';
+import { ScreenReaderOnly } from 'src/components/matchbox';
```
Comment (nicklemmon) [inline src/components/iconButton/IconButton.js]: Just changed this - the rest is Prettier.

File: src/pages/templates/components/SavedIndicator.js [hunk 3]
```diff
@@ -1,19 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { ScreenReaderOnly } from '@sparkpost/matchbox';
+import { ScreenReaderOnly } from 'src/components/matchbox';
```
Comment (nicklemmon) [inline src/pages/templates/components/SavedIndicator.js]: Just changed this, the rest is Prettier.

File: src/components/hibana/DesktopNavigation.js [hunk 4]
```diff
@@ -28,7 +28,7 @@ export default function DesktopNavigation(props) {
 
           <nav className={styles.PrimaryNav} aria-labelledby="primary-nav-heading">
             {/* Visually hidden headings to help guide screen reader users */}
-            <ScreenReaderOnly>
+            <ScreenReaderOnly as="div">
```
Comment (bkemper) [inline src/components/hibana/DesktopNavigation.js]: Hmm, I thought h2 would be the as.

```
<ScreenReaderOnly as="h2">
  Main
</ScreenReaderOnly>
```

Comment (nicklemmon) [reply src/components/hibana/DesktopNavigation.js]: 🤷‍♂ Either way works!

**Comment by bkemper** [comment]
Look like unit tests are failing.

Screen Shot 2020-03-26 at 10 38 34 AM

**Comment by nicklemmon** [comment]
> Look like unit tests are failing.
> 
> Screen Shot 2020-03-26 at 10 38 34 AM

Just addressed it - we can wait on Travis and then deploy whenever.

**Comment by bkemper** [comment]
@nicklemmon 😭 

Screen Shot 2020-03-26 at 12 37 45 PM

**Pull request merged**